editor/css/ketcher-extras.css

```diff
@@ -0,0 +1,344 @@
+/**
+ * Copyright 2020 Open Reaction Database Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Styling for properly displaying Ketcher in the reaction page. 
+ * Mainly derived from Bootstrap v4.5.0 (https://getbootstrap.com/).
+ */
```

Comment:
Can you say something about why you're using Bootstrap here?

Is there something tricky about displaying Ketcher? Could there be a simpler way?

Reply:
We decided on displaying Ketcher through a modal. Showing an iframe in the middle of the page, hiding the background, etc. is doable with our own CSS, but we figured it'd be okay to use what Bootstrap has.

(Less importantly, using Bootstrap also allowed us to use their spinners too.)

Reply:
I see. This could be fine. Maybe in the meeting you can just explain the decision.

Reply:
I think you need to create a separate file for the boostrap-only things, and put their license header at the top:

https://github.com/twbs/bootstrap/blob/89dc9750ec029a1479cba87f27ab936df6edb681/js/src/alert.js#L1-L6

Then put your own CSS in another file with our usual ORD header.

Reply:
Sounds good, thanks. Should I also remove the comments I wrote? (these: 

![image](https://user-images.githubusercontent.com/14008996/89354621-298bdd00-d66e-11ea-9084-1fd78dcc300d.png)

Reply:
I think it's fine to put a link to the source file, but I would leave out the other comments. Make sure you clearly indicate any changes you make to their CSS. If it's just copy-paste of select elements (and no changes), you should indicate that in the comments below the header as well.

Reply:
I'd use the header from that source file directly:

/*!
 * Bootstrap v4.5.0 (https://getbootstrap.com/)
 * Copyright 2011-2020 The Bootstrap Authors
 * Copyright 2011-2020 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 */

Reply:
How does this look?

![image](https://user-images.githubusercontent.com/14008996/89355154-86d45e00-d66f-11ea-9196-39e13da5723c.png)

Reply:
Can you use the header from the original file?

Reply:
@skearnes just saw your comment about the original header (not sure why I didn't earlier, sorry). Switched to that; anything else? 

![image](https://user-images.githubusercontent.com/14008996/89356144-ea5f8b00-d671-11ea-9c9a-8d792cfa8d09.png)

Reply:
LGTM; thanks.